azure/group.sh

```diff
@@ -7,8 +7,21 @@ fn azure_group_create(name, location) {
 	azure group create --name $name --location $location
 }
 
+# azure_group_get_id get a `resource group` id.
+# `name` is the resource group name
+#
+# This function return resource group id
+fn azure_group_get_id(name) {
+        resp <= (
+                azure group show --name $name --json
```

Comment:
Maybe should be good to leave this function as is, and add another one to just check if the resource group exists (maybe called `azure_group_check`).